packages/components/src/border-box-control/stories/index.tsx

```diff
@@ -35,72 +52,57 @@ const colors = [
 	{ name: 'Yellow 40', color: '#bd8600' },
 ];
 
-export default {
-	title: 'Components (Experimental)/BorderBoxControl',
-	component: BorderBoxControl,
-	parameters: {
-		knobs: { disable: false },
-	},
-};
+const Template: ComponentStory< typeof BorderBoxControl > = ( props ) => {
+	const { onChange, ...otherProps } = props;
+	const [ borders, setBorders ] = useState< AnyBorder >();
 
-const _default = ( props ) => {
-	const { defaultBorder } = props;
-	const [ borders, setBorders ] = useState( defaultBorder );
-
-	useEffect( () => setBorders( defaultBorder ), [ defaultBorder ] );
+	const onChangeMerged: ComponentProps<
+		typeof BorderBoxControl
+	>[ 'onChange' ] = ( newBorders ) => {
+		setBorders( newBorders );
+		onChange( newBorders );
+	};
 
 	return (
 		<SlotFillProvider>
-			<WrapperView>
+			<div style={ { maxWidth: '248px', padding: '16px' } }>
 				<BorderBoxControl
-					colors={ colors }
-					label="Borders"
-					onChange={ ( newBorders ) => setBorders( newBorders ) }
+					{ ...otherProps }
+					onChange={ onChangeMerged }
 					value={ borders }
-					{ ...props }
 				/>
-			</WrapperView>
-			<Separator />
-			<HelpText>
+			</div>
+			<hr
+				style={ {
+					marginTop: '100px',
+					borderColor: '#ddd',
+					borderStyle: 'solid',
+					borderBottom: 'none',
+				} }
+			/>
+			<p style={ { color: '#aaa', fontSize: '0.9em' } }>
 				The BorderBoxControl is intended to be used within a component
 				that will provide reset controls. The button below is only for
 				convenience.
-			</HelpText>
-			<Button variant="primary" onClick={ () => setBorders( undefined ) }>
+			</p>
+			<Button
+				variant="primary"
+				onClick={ () => onChangeMerged( undefined ) }
+			>
 				Reset
 			</Button>
+			{ /* @ts-expect-error Ignore until Popover is converted to TS */ }
```

Comment:
Nit: Popover is actually converted to TypeScript, but its slot is not
 
```suggestion
			{ /* @ts-expect-error Ignore until Popover.Slot gets typed for the Popover component */ }
```

Reply:
Thanks. It's good to be accurate. I've fixed this here and in the `BorderControl` where I copied it from 🙂 

https://github.com/WordPress/gutenberg/pull/45002/commits/8277b809a6475513c602d1502f3a1062f5667a84